HousingManagementSystemApi/Gateways/AddressesHttpGateway.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections.Generic;
```

Comment:
This class and file have been renamed (to `HousingSearchGateway`) and doesn't need to be reintroduced.

Please delete this file.